### ISSUE TITLE
NOTICE: ALL FEATURE BRANCHES MOVED TO ANOTHER REPO

### ISSUE THREAD

**Issue by ctrlplusb** [ctrlplusb/react-universally#570]
https://github.com/ctrlplusb/react-universally-archive

I apologise if this upsets anyone.

I may move a few choice ones back into this repo after we have v14 sorted.

**Comment by sergiokopplin** [comment]
Hey @ctrlplusb, it seems a good approach. Any reasons on that?

**Comment by birkir** [comment]
Great! More focus on what really matters, the core foundation!

**Comment by ctrlplusb** [comment]
Yeah, we have a good momentum going again now - I didn't want to risk the maintenance of the feature branches scare people off.
